### PR TITLE
feat: form validation

### PR DESCRIPTION
**Pull request by ferrarafer**
- Added PaymentCaptureView
- Added validation for `cardExpiry` and `cardCvv` fields
- Updated InputField to show validation message as errorText
- Updated AuthGuard redirection route to UnknownView
- Fixed navigateToHome function on UnknownView
- Added a few links to navigate through app during development

### CODE REVIEW

File: lib/ui/views/payment_capture/payment_capture_view.desktop.dart [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            ),
+            verticalSpaceMedium,
+            if (viewModel.showValidationMessage) ...[
+              Text(
+                viewModel.validationMessage!,
+                style: ktsBodyLarge.copyWith(color: Colors.redAccent),
+              ),
+              verticalSpaceLarge,
+            ],
+            InputField(
```
Comment (FilledStacks) [inline lib/ui/views/payment_capture/payment_capture_view.desktop.dart:51]: We should add `TextInputFormatters` here. It's a quick win and makes the input much better. It'll be great to show that functionality to users as well. 

In venu we have the text input formatters.

**Review by FilledStacks — CHANGES_REQUESTED** [review]
I'm happy with the validation logic. Shows what we need to. I think one extra thing we can add is text input formatting. I left a comment where we want to add that.

File: lib/ui/views/payment_capture/payment_capture_view.desktop.dart (hunk 1 above)
Comment (ferrarafer) [reply lib/ui/views/payment_capture/payment_capture_view.desktop.dart:51]: Yes, I agree. I though about it but I didn't want it to add it before talking with you. I will add it when I wake up on Friday 😅 .

**Review by FilledStacks — APPROVED** [review]
🔥

**Pull request merged**